spec/features/login_spec.rb

```diff
@@ -0,0 +1,51 @@
+feature "Sign in" do
+  scenario "a user can sign in with correct email and password" do
+    user = User.create(name: "Ruby", username: "rb", email: 'test@ruby.com', password: 'password')
+
+    visit '/users/login'
+    fill_in(:email, with: 'test@ruby.com')
+    fill_in(:password, with: 'password')
+    click_button('Sign in')
+
+    expect(page).to have_content 'Welcome, Ruby'
+  end
+   
+  scenario "a user can not sign in if not already registered or wrong email" do
+    visit '/users/login'
+    fill_in(:email, with: 'notregistered@email.com')
+    fill_in(:password, with: 'password')
+    click_button('Sign in')
+
```

Comment:
Couldn't you have raised an error here in the test instead? Not sure if possible, just asking.